node/risk-app/server/plugins/views.js

```diff
@@ -44,14 +44,14 @@ module.exports = {
       G4AnalyticsAccount,
       GTagManagerId,
       appStage: config.env,
-      floodWarningsUrl: config.floodWarningsUrl,
+      floodWarningsUrl: config.floodWarningsUrl ? config.floodWarningsUrl : process.env.FLOOD_WARNINGS_URL,
       phase: 'beta',
       feedback: true,
-      siteUrl: config.floodRiskUrl,
+      siteUrl: config.floodRiskUrl ? config.floodRiskUrl : process.env.FLOOD_RISK_URL,
       ogDescription: 'Check your risk of flooding and use flood risk maps',
       noIndex: true, // stop robot crawl by default
-      friendlyCaptchaEnabled: config.friendlyCaptchaEnabled,
-      friendlyCaptchaSiteKey: config.friendlyCaptchaSiteKey
+      friendlyCaptchaEnabled: config.friendlyCaptchaEnabled ? config.friendlyCaptchaEnabled : process.env.FRIENDLY_CAPTCHA_ENABLED,
+      friendlyCaptchaSiteKey: config.friendlyCaptchaSiteKey ? config.friendlyCaptchaSiteKey : process.env.FRIENDLY_CAPTCHA_SITE_KEY
```

Comment:
All reading of environment variables should be done in the config module. I'm unsure why this change to check `process.env` is here, but this is the wrong place for it. This all needs to be done by the config module so that it can be mocked in tests, and there is a single place for reading validating config values.